src/services/dashboard.js

```diff
@@ -133,7 +137,9 @@ export async function queryTocResult(params) {
     selectedDateRange
   )}/_search?q=run_data_parent:"${id}"`;
 
-  const endpoint = `${endpoints.pbench_server}/elasticsearch`;
+  const endpoint = MOCK_UI
```

Comment:
As a first step, we'll define custom endpoints that have not yet been defined in `pbench_server`. Eventually, these will be replaced with their appropriate endpoints. These mocked endpoints are only available when the `MOCK_UI` env variable is set to True.